Makefile

```diff
@@ -11,7 +11,6 @@ deps-update:
 	pip3 freeze -r requirements-top-level.txt > requirements.txt
 
 lint:
-	yapf $(analysis_directories) --diff --parallel --recursive --style google
```

Comment:
This is the step we don't want enforced for external contributors. We could run it in a separate CI check, but why pay for a check we won't enforce?

Instead, in the future, we can add auto-formatting like we have in the panther repo